packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -246,8 +246,14 @@ export class FeatureDevController {
     private async onApproachGeneration(session: Session, message: string, tabID: string) {
         await session.preloader(message)
 
+        this.messenger.sendAnswer({
+            type: 'answer',
+            tabID,
+            message: 'Ok, let me create a plan. This may take a few minutes.',
```

Comment:
This is also user facing text, right?


Let's extract this also to the shared place